app/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,22 @@
     </string-array>
     <string name="location">Please pick your location!</string>
 
+    <string-array name="from_currencies_array">
+        <item>CAD</item>
+        <item>EUR</item>
+        <item>GBP</item>
+        <item>INR</item>
+        <item>USD</item>
+    </string-array>
+
+    <string-array name="to_currencies_array">
```

Comment:
why not reuse the same one?

Reply:
I made two of them so we can specify different orders for the base currency and destination currency.